runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/cache/KafkaCacheCursorFactory.java

```diff
@@ -408,10 +413,10 @@ public int next(
             }
 
             @Override
-            public boolean test(
+            public long test(
                 KafkaCacheEntryFW cacheEntry)
             {
-                return cacheEntry != null;
+                return cacheEntry != null ? 1L : 0L;
```

Comment:
Does this need to be `-1` to generate the default, non-zero value for `None` filter?